### PR TITLE
Tool for evaluating trigger bindings.

### PR DESCRIPTION
**Pull request by wlynch**
# Changes

This tools allows for evaluating TriggerBindings against an HTTP
request. This is a convenience tool to allow people to test binding
expressions.

This is the first step of this tool - other useful features like
evaluating triggers -> resources or reading HTTP requests from stdin can
come in future changes.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Reviewer Notes

I have no idea what to name this binary. If you have any ideas, let me know!

# Release Notes

```
Adds `binding-eval` binary for evaluating TriggerBindings from HTTP requests.
```

### CODE REVIEW

File: docs/triggerbindings.md [hunk 1]
```diff
@@ -132,3 +134,45 @@ spec:
       template:
         name: pipeline-template
 ```
+
+## Debugging
+
+### Evaluating TriggerBindings
+
+As a convenience, the `binding-eval` tool allows you to evaluate TriggerBindings
```
Comment (vtereso) [inline docs/triggerbindings.md]: Maybe just an extra line specifying to `go install`?

**Comment by wlynch** [comment]
Totally forgot I had this open 😅 

Rebased and added one small fix.

**Pull request merged**